root.json

```diff
@@ -3,14 +3,15 @@
     "Airsonic Advanced": "airsonic-advanced.json",
     "Booksonic": "booksonic.json",
     "Calibre": "calibre.json",
-	"Collabora Online": "collabora-online.json",
+    "Collabora Online": "collabora-online.json",
     "COPS": "cops.json",
     "Deluge": "deluge.json",
     "Dropbox": "dropbox.json",
     "DuckDNS": "duckdns.json",
     "Duplicati - by Linuxserver.io": "duplicati-lsio.json",
     "ecoDMS 18.09": "ecodms18-09.json",
     "Emby server": "embyserver.json",
+    "ESPHome": "ESPHome.json"
```

Comment:
```suggestion
    "ESPHome": "ESPHome.json",
```

@Hooverdan96 I think we are missing a comma here. Otherwise we get a parse error on our Rockon index file. I.e. when testing locally with my production mimic setup of using build.lan in loo of PRODUCTION I get:

> Houston, we've had a problem.
> (Error while processing remote metastore at (http://build.lan/rockons/root.json).). Lower level exception: (Expecting ',' delimiter: line 15 column 5 (char 485)). 

```
Traceback (most recent call last):
  File "/opt/rockstor/.venv/lib/python3.11/site-packages/requests/models.py", line 974, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
               ^^^^^^^^^^^^^^^^^^^^^^
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 15 column 5 (char 485)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/rockstor/src/rockstor/rest_framework_custom/generic_view.py", line 40, in _handle_exception
    yield
  File "/opt/rockstor/src/rockstor/storageadmin/views/rockon.py", line 490, in _get_available
    root = response.json()
           ^^^^^^^^^^^^^^^
  File "/opt/rockstor/.venv/lib/python3.11/site-packages/requests/models.py", line 978, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting ',' delimiter: line 15 column 5 (char 485)
```